src/pydocstyle.py

```diff
@@ -140,13 +144,17 @@ def is_empty_or_comment(line):
         return ''.join(reversed(list(filtered_src)))
 
     def __str__(self):
-        return 'in %s %s `%s`' % (self._publicity, self._human, self.name)
+        out = 'in %s %s `%s`' % (self._publicity, self._human, self.name)
```

Comment:
I'll only hold you to lines you changed, but if you feel like going over the code, I'd be grateful.